superset-frontend/src/SqlLab/components/SaveDatasetModal/index.tsx

```diff
@@ -100,19 +100,23 @@ const Styles = styled.div`
     margin: 0 8px;
   }
   .sdm-input {
-    margin-left: 45px;
+    margin-left: 40px;
```

Comment:
Can we use the `gridUnit` here? This whole piece style is not using the gridUnit for margins and paddings as it should, can you update it, please?